Network/dnsswitcher.1d.sh

```diff
@@ -1,15 +1,20 @@
 #!/usr/bin/env bash
 #
 # DNS Switcher
-# BitBar plugin
+# The list of DNS options should be defined on this file
 #
-# by M Saiqul Haq
+# <bitbar.title>DNS Switcher</bitbar.title>
+# <bitbar.version>v1.4</bitbar.version>
+# <bitbar.author>M Saiqul Haq</bitbar.author>
+# <bitbar.author.github>saiqulhaq</bitbar.author.github>
+# <bitbar.desc>Switch DNS to your defined DNS options</bitbar.desc>
```

Comment:
Yes exactly! The metadata is `<bitbar.image>`. [See here](https://github.com/matryer/bitbar#metadata)